### PR TITLE
bump version for terraform-aws-iam-system-user

### PR DESCRIPTION
**Pull request by maximmi**
## What
* bump version of terraform-aws-iam-system-user module
* policy removed from module attribute

## Why
* chain of updates started from [terraform-aws-iam-system-user](https://github.com/cloudposse/terraform-aws-iam-system-user/pull/13)
* requirements changes in new `terraform-aws-iam-system-user` module

## ToDo
- [x] update `terraform-aws-iam-system-user` module version using tag when it will be released

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -16,5 +16,4 @@ module "s3_user" {
   enabled       = "${var.enabled}"
   force_destroy = "${var.force_destroy}"
   path          = "${var.path}"
-  policy        = "${data.aws_iam_policy_document.default.json}"
 }
```
Comment (goruha) [inline main.tf]: Add here `user policy attach`

**Comment by maximmi** [comment]
> re-enable `policy = "${data.aws_iam_policy_document.default.json}"`

Igor removed this in release 0.3.0 [commit](https://github.com/cloudposse/terraform-aws-iam-system-user/commit/1ceb93a3b6ec251d0569509e136c97286b72884a)

File: main.tf (hunk 1 above)
Comment (maximmi) [reply main.tf]: done

**Comment by joshmyers** [comment]
Given that @aknysh approved https://github.com/cloudposse/terraform-aws-iam-system-user/commit/1ceb93a3b6ec251d0569509e136c97286b72884a I assume it is OK to externalise the policy and anything that uses this will need to change when upgrading to latest release.

**Pull request merged**